### PR TITLE
2 create basic app

### LINKED ISSUE #2: Create Basic App
Acceptance Criteria: App is deployable with basic structure.

### PR DESCRIPTION
**Pull request by devkdo**
Changes made: Added 2 screens using React Native Expo.
- Login Screen
- Dashboard Screen

Closes Issue https://github.com/devkdo/momentu/issues/2

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { View } from 'react-native';
+import LoginScreen from './app/screens/LoginScreen';
```
Comment (Blatantness) [inline App.js:3]: Do you need this?

File: app.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "expo": {
+    "name": "momentu",
+    "slug": "momentu",
```
Comment (Blatantness) [inline app.json:4]: Do need?

File: App.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { View } from 'react-native';
```
Comment (Blatantness) [inline App.js:2]: Do you need this?

File: app/components/SquareButton.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { Button, StyleSheet, Text, View } from 'react-native'
+
+export default function SquareButton() {
+  const [count, setCount] = useState(0)
+
+  return (
+    <View style={styles.container}>
+      <Button
+        title={'Press me!'}
```
Comment (Blatantness) [inline app/components/SquareButton.js:10]: What is this for?

File: app/containers/UserProfile.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Image, StyleSheet } from 'react-native'
+
+export default function UserProfile() {
```
Comment (Blatantness) [inline app/containers/UserProfile.js:4]: What will this be used for?

File: babel.config.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
```
Comment (Blatantness) [inline babel.config.js:4]: What is babel?

**Review by Blatantness — COMMENTED** [review]
gg

File: App.js (hunk 1 above)
Comment (devkdo) [reply App.js:3]: No I don't need it . But I'm keeping it 🤣

Comment (devkdo) [reply App.js:3]: This screen should be in the app, but I haven't put it in yet. Next issue #12

File: app.json (hunk 2 above)
Comment (devkdo) [reply app.json:4]: We need a slug #10

File: babel.config.js (hunk 6 above)
Comment (devkdo) [reply babel.config.js:4]: adds support for decorators, tree-shaking web packages, and loading font icons

**Pull request merged**